### PR TITLE
doc: add installation instructions to README

### PR DESCRIPTION
**Pull request by guisp03**
This pull request addresses issue #77 by adding a new section to the README.md with installation instructions for the game. The instructions include:
- Prerequisites, such as Python 3 and the Pygame engine.
- Step-by-step guidance for installing Pygame, covering both pip and alternative package managers (e.g., pacman for Arch-based systems).
- Instructions on how to clone the repository, navigate to the project folder, and run the game using the coral.py file.

These additions aim to provide a clear and accessible guide for new developers or users unfamiliar with Python and Pygame, ensuring an easier setup process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,4 +40,51 @@
...
+ See `docs/CONTRIBUTING.md` for further information.
+
+ Installation
+ ------------------------------
+
+ Follow these steps to set up and run the Coral Snake Game:
+
+ ### Prerequisites
+ 
+ - Python 3: Make sure you have Python 3 installed on your machine. You can download it from the official Python website.
```
Comment (ImLusca) [inline README.md:52]: Add python download link using `download it from the official Python [website](https://www.python.org/downloads/)`

Comment (ImLusca) [inline README.md:52]: Add the apt install command for linux users

File: README.md [hunk 2]
```diff
@@ -40,4 +40,51 @@
...
+ - Pygame: The game requires the Pygame library, which can be installed using `pip` or other package managers like `pacman` on Arch Linux.
+
+ ### Installation Steps
+ 
+ 1. Clone the repository:
+
+ First, clone the Coral repository to your local machine by running the following command in your terminal:
+
+ ```bash
+ git clone https://github.com/your-username/coral.git
```
Comment (ImLusca) [inline README.md:63]: github http auth is deprecated. Use SSH instead

**Review by ImLusca — CHANGES_REQUESTED** [review]

**Comment by guisp03** [comment]
Hi @ImLusca , I’ve made the requested changes:
- Added the Python download link for Windows/Mac users.
- Included the `apt install` command for Linux users with a note on distribution differences.
- Updated the clone command to SSH.
Let me know if there’s anything else to adjust. Thanks for the feedback!

**Review by samuks123 — APPROVED** [review]
All the requested changes have been resolved

**Pull request merged**